### PR TITLE
Right Sidebar preview improvement

### PR DESCRIPTION
**Pull request by sacarvy**
## Changes

- Docs Right Sidebar
- Highlighting

## Docs
It's Public Docs Yes

Request for a preview from @jonathantneal ,

issue PR #1819

### CODE REVIEW

**Comment by matthewp** [comment]
If you rebase with `main` this should pass again now.

File: docs/src/layouts/MainLayout.astro [hunk 1]
```diff
@@ -5,21 +5,24 @@ import Header from '../components/Header/Header.astro';
 import Footer from '../components/Footer/Footer.astro';
 import PageContent from '../components/PageContent/PageContent.astro';
 import LeftSidebar from '../components/LeftSidebar/LeftSidebar.astro';
-import RightSidebar from '../components/RightSidebar/RightSidebar.astro';
+// import RightSidebar from '../components/RightSidebar/RightSidebar.astro';
+import RightSidebarV2 from '../components/RightSidebarV2/index.astro';
```
Comment (jonathantneal) [inline docs/src/layouts/MainLayout.astro:9]:
```suggestion
import RightSidebarV2 from '../components/RightSideBarV2/index.astro';
```

**Comment by natemoo-re** [comment]
Hey @sacarvy would it be possible to revert `6a970d2` and `6009724`? Changesets should not be included in PRs like these.

**Comment by natemoo-re** [comment]
Following up on my last comment!

I discussed this PR with the team. We really appreciate all the effort you put into this, but we're concerned due to the number of bugs we encountered the last time we tried to implement this.

I'd personally really like to see this feature added back to our site, but it might make sense to be built as a third-party component first. Would you be interested in working on an Astro component that implements this feature to collect some more real-world data before bringing it into our docs site?

**Pull request closed without merging**